packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -904,7 +904,12 @@ class _CupertinoModalPopupRoute<T> extends PopupRoute<T> {
 ///
 /// The `useRootNavigator` argument is used to determine whether to push the
 /// popup to the [Navigator] furthest from or nearest to the given `context`. It
-/// is `false` by default.
+/// is `true` by default.
+///
+/// The `barrierColor` argument is used to determine use barrier [Color] to use.
+/// If [Color] passed is fully transparent (it's Alpha channel is equal 0) then
```

Comment:
Oh we don't need to mix these things. If you want to express the 2 concepts (whether there is one, and if there is one, what color it is), just let there be 2 parameters like material/dialog.dart's barrierDismissible and barrierColor.